src/App.js

```diff
@@ -1,25 +1,88 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import "./App.css";
+import LoginScreen from "./components/LoginScreen/LoginScreen";
+import Dashboard from "./components/Dashboard/Dashboard";
+import { Component } from "react";
+import { BrowserRouter as Router, Link, Switch, Route } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isLoggedIn: false,
+      flightCodes: [],
+      squadronData: [],
+    };
+  }
+
+  async componentDidMount() {
+    const response = await fetch(
+      "https://us-central1-cc-dashboard-afb15.cloudfunctions.net/getFlightCodes"
+    );
+    const newFlightCodes = await response.json();
+    this.setState({ flightCodes: newFlightCodes });
+
+    let output = [];
+    await Promise.all(
+      await this.state.flightCodes.flights.map(async (code) => {
+        const airmenResponse = await fetch(
+          `https://us-central1-cc-dashboard-afb15.cloudfunctions.net/getAirmen/${code}`
+        );
+        const airmenData = await airmenResponse.json();
+        output.push({ flightName: code, flightData: airmenData });
+      })
+    );
```

Comment:
This can likely be simplified a bit.  There are a few things that could be refactored - we could map the promises to the output array and use Promise.all on that array of promises instead.

We could also probably consider encapsulating all of the fetch requests into their own file for export as methods.  The app can still make use of them in the same sort of way, but extracting and encapsulating that activity can create more flexibility and modularity in how we can work with our code.